### DISCUSSION TITLE
Welcome to homebridge-wyze-connected-home-op Discussions!

### DISCUSSION THREAD

**Discussion by ndejong5** [ndejong5/homebridge-wyze-connected-home-op#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wveilleux** [comment]
Is there a plan to integrate Wyze Smart Door Lock ?

**Comment by boothmusic** [comment]
Any plan to add integration for WYZE switch? It’d Be great for those of us with smart lights to run the switch in “smart mode” and leave constant power to Lifx bulbs. Then have Homebridge recognize that a button was pushed then have an automation to change the status of specific lights
